.storybook/main.js

```diff
@@ -7,24 +7,20 @@ const configFactory = require('../config/webpack.config');
 const sassRegex = /\.(scss|sass)$/;
 
 module.exports = {
-  "stories": [
-    "../src/**/*.stories.mdx",
-    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+    'storybook-addon-themes',
+    'storybook-react-intl',
   ],
-  "addons": [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials",
-    "@storybook/addon-interactions",
-    "storybook-addon-themes",
-    "storybook-addon-mock",
```

Comment:
When you showed my your story book it had an empty mocking tab in the controls. Removing this addon removes that, right?

Reply:
Correct!